content/en/docs/reference/access-authn-authz/service-accounts-admin.md

```diff
@@ -11,7 +11,7 @@ weight: 50
 
 <!-- overview -->
 This is a Cluster Administrator guide to service accounts. It assumes knowledge of
-the [User Guide to Service Accounts](/docs/user-guide/service-accounts).
+the [User Guide to Service Accounts](/docs/tasks/configure-pod-container/configure-service-account/).
```

Comment:
nit: Could change text to something like:

This is a cluster administrator's guide to service accounts. You should be familiar with
[configuring Kubernetes Service accounts](/docs/tasks/configure-pod-container/configure-service-account/).

Reply:
suggestions taken.